src/Native/Unix/configure.cmake

```diff
@@ -676,6 +707,16 @@ endif()
 
 set (CMAKE_REQUIRED_LIBRARIES)
 
+check_cxx_source_compiles(
+    "
+    #include <sys/inotify.h>
+    int main()
+    {
+        uint32_t mask = IN_EXCL_UNLINK;   
```

Comment:
Oops, thanks for spotting it. Interestingly enough, 10 other check_cxx_source_compiles
 tests in this file are also missing `return something` at the end of main. So I am adding it just in case a future more strict compiler would refuse to compile that.